mitodl/ocw-studio
# update ocw_import to support ocw-to-hugo 1.30.3

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/598

#### What's this PR do?
In https://github.com/mitodl/ocw-to-hugo/pull/350 and some subsequent PR's, we began altering `course.-json` and created `course_legacy.json` to preserve the state of `course.json` before modifications started being made to conform to the structure of the `ocw-course` starter config at https://github.com/mitodl/ocw-to-hugo/pull/350.  This PR adjusts the `ocw_import` code to use this file instead and updates the import process to support `ocw-to-hugo` 1.30.3` which includes changes to topics as well as changes to the structure of content files from PR's like https://github.com/mitodl/ocw-to-hugo/pull/337 that change the location of resource markdown.

#### How should this be manually tested?
 - Set AWS credentials for `open-learning-course-data-rc` (or production) and make sure you are set up to use `ocw_import` by having an `ocw-www` site based off of https://github.com/mitodl/ocw-hugo-projects/blob/main/ocw-www/ocw-studio.yaml
 - Spin up `ocw-studio` locally
 - Run `docker-compose run web ./manage.py import_ocw_course_sites -b ocw-to-hugo-output-rc --limit 1`
 - Verify that you have a site at http://localhost:8043/sites/1-00-introduction-to-computers-and-engineering-problem-solving-spring-2012/ successfully imported.  It should have pages, resources and instructors in the metadata.
